src/core/a-entity.js

```diff
@@ -145,12 +145,18 @@ var proto = Object.create(ANode.prototype, {
 
   setObject3D: {
     value: function (type, obj) {
+      var self = this;
       var oldObj = this.object3DMap[type];
       if (oldObj) { this.object3D.remove(oldObj); }
       if (obj instanceof THREE.Object3D) {
-        obj.el = this;
+        obj.el = self;
         this.object3D.add(obj);
       }
+      if (obj instanceof THREE.Group && obj.children.length) {
+        obj.traverse(function bindEl (child) {
```

Comment:
From what I can see inside Three's [OBJLoader](https://github.com/mrdoob/three.js/blob/44844c77f491e3115c3473f0237c25d6b1146d32/examples/js/loaders/OBJLoader.js#L163), all groups and objects are throw into a flat list. So this code would work with any OBJ borne mesh. I only worry about other formats; but if that's not the case, this could work:

```
if (obj instanceof THREE.Object3D) {
  obj.el = self;
  this.object3D.add(obj);
  if (obj.children.length) {
    obj.traverse(function bindEl (child) {
      child.el = self;
    });
  }
}
```